docs/excel/excel-add-ins-tables.md

```diff
@@ -160,6 +160,37 @@ Excel.run(function (context) {
 
 ![Table with new calculated column in Excel](../images/excel-tables-add-calculated-column.png)
 
+## Resize a table (preview) (online-only)
+
+> [!NOTE]
+> The `Table.resize` method is currently available in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)].
+> 
+
+> [!NOTE]
+> The `Table.resize` method is also currently available in ExcelApiOnline 1.1. To learn more, see [Excel JavaScript API online-only requirement set](../reference/requirement-sets/excel-api-online-requirement-set.md).
+
+Your add-in can resize a table without adding data to the table. To resize a table, use the [resize](/javascript/api/excel/excel.table#resize_newRange_) method. The following code sample shows how to resize a table and set the new range of the table to **A1:D20**.
```

Comment:
Something to consider for a future refactoring (not this PR):
We use this "Your add-in can" phrasing a lot. It's not the most exciting start to a sentence, especially going section by section. Plus, the style guide advises avoiding "can". Maybe we can find a way to improve this.

Reply:
I will reflect on this. I have relied on this phrasing because it's common in our docs, but I agree with your point!